dingo-expr/runtime/src/main/java/io/dingodb/expr/runtime/evaluator/type/TypeEvaluators.java

```diff
@@ -171,34 +171,13 @@ static String stringType(@Nonnull Object value) {
             return Time2StringUtils.convertTime2String((java.sql.Time) value);
         } else if (value instanceof java.sql.Timestamp) {
             return Time2StringUtils.convertTimeStamp2String((java.sql.Timestamp) value, "yyyy-MM-dd HH:mm:ss");
-        } else if (value instanceof Long) {
-            return stringType((Long) value);
         } else {
             return value.toString();
         }
     }
 
     @Evaluators.Base(StringEvaluator.class)
     static String stringType(@Nonnull Long value) {
```

Comment:
new function such as `stringType(Timestamp value)` `stringType(Date value)` is needed?

Reply:
`stringType(Long )` should be discard, as long can convert to string